### PR TITLE
Used different CNN hyperparameters + loss + adam LR

### PR DESCRIPTION
**Pull request by QFred**
Created new CNN model based on other works using tanh activation, changed loss function from Huber to Categorical Crossentropy, and increased learning rate of adam from 0.00025 to 0.0005

### CODE REVIEW

**Comment by QFred** [comment]
Reverted to original new-start and added more comments

**Comment by QFred** [comment]
Updated more comments in main.py
